DiscordBot/Commands/EventCommands.cs

```diff
@@ -19,34 +20,109 @@ internal class EventCommands
         };
 
         [Command("event")]
+        [Description("Initiates the wizard for event-related actions")]
```

Comment:
How does this description get used? 🙂

Reply:
There is a built-in help command that lists each available command and the description.

Reply:
I see, thanks 🙂